examples/ssh/dell_idrac_phisical_disks_infos.json

```diff
@@ -0,0 +1,45 @@
+{
+    "logo": "dell.svg",
+    "version": "1.0.1",
+    "name": "Dell iDRAC Physical Disk Monitoring",
+    "description": "Monitors the operational status of physical disks on a Dell server with iDRAC",
+    "category": "Hardware Integration",
+    "requirements": {
+        "credentials": false,
+        "sandbox_version": "1.12",
+        "others": [
+            "SSH Enabled",
+            "The keyboard_interactive option can be true or false depending on the iDRAC version"
```

Comment:
Its sent explicitly in the SSH send.
Does the driver work with it disabled for this claim to be true?